core/parachain/validator/impl/parachain_processor.cpp

```diff
@@ -250,6 +250,7 @@ namespace kagome::parachain {
   }
 
   bool ParachainProcessorImpl::start() {
+    thread_pool_->handler().start();
```

Comment:
Who should `start` (`ThreadPool` is constructor argument, which means other components may use it)?

Reply:
`start` in `ThreadHandler` constructor, or `ThreadPool { io_context }` not `ThreadPool { ThreadHandler }`